.github/dependabot.yml

```diff
@@ -7,8 +7,12 @@ updates:
     schedule:
       # Check for updates to GitHub Actions every week
       interval: "weekly"
+    commit-message:
+      prefix: "chore"
   - package-ecosystem: "swift"
     directory: "/"
     schedule:
       interval: "weekly"
-    open-pull-requests-limit: 5
\ No newline at end of file
+    commit-message:
+      prefix: "chore"
+    open-pull-requests-limit: 5
```

Comment:
Can we make this "build"? https://github.com/Oliver-Binns/Versioning/pull/13

Reply:
have changed this now to build